widgets/src/Makefile.am

```diff
@@ -61,7 +61,7 @@ lib_LTLIBRARIES = libAnacondaWidgets.la
 libAnacondaWidgets_la_CFLAGS = $(GTK_CFLAGS) $(GLADEUI_CFLAGS) $(LIBXKLAVIER_CFLAGS) -Wall -g\
 			       -DWIDGETS_DATADIR=$(WIDGETSDATA)
 libAnacondaWidgets_la_LIBADD = $(GTK_LIBS) $(GLADEUI_LIBS) $(LIBXKLAVIER_LIBS)
-libAnacondaWidgets_la_LDFLAGS = $(LTLIBINTL) -version-info 3:1:1
+libAnacondaWidgets_la_LDFLAGS = $(LTLIBINTL) -version-info 3:2:1
```

Comment:
According to some slides about Autotools I have I think this change should be to ``4:0:2``

![library_version_changes](https://cloud.githubusercontent.com/assets/1044278/12742342/23584afe-c985-11e5-985b-cf15505ab47e.png)